test/navbar_router_test.dart

```diff
@@ -1793,4 +1795,103 @@ void main() {
     await tester.pumpAndSettle();
     expect((navItems[2].selectedIcon as Icon).color, Colors.green);
   });
+
+  group('Swipeable extra test', () {
+    testWidgets('Should not be able to drag on center', (WidgetTester tester) async {
```

Comment:
By default the draggable area should take the width of available view including center,